app/components/polls/questions/answers_component.rb

```diff
@@ -10,16 +10,18 @@ def already_answered?(answer)
     user_answers.find_by(answer: answer.title).present?
```

Comment:
Maybe we can now reuse the new method :thinking:.

```suggestion
    user_answer(answer).present?
```